.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+# Contributing to Garden
+
+Keen to contribute to Garden? We're stoked to have you join us. You may
+find that opening an
+[issue](https://github.com/zendeskgarden/website/issues) is the best
+way to get a conversation started. When you're ready to submit a pull
+request, follow the [steps](#pull-request-workflow) below. We follow a
+[code of conduct](CODE_OF_CONDUCT.md) as our guide for community
+behavior.
+
+## Development Workflow
+
+Before you start, be sure [yarn](https://yarnpkg.com/en/) is installed
+on your system. After you clone this repo, run `yarn` to install
+dependencies needed for development. After installation, the following
+commands are available:
```

Comment:
Just double-checking here. Once we enable the Abstract layer sourcing it will be impossible for anyone outside of the Zendesk org to build the site locally. Should we call that out here?

Reply:
Not at this time. Let's see how things evolve. I don't envision external involvement beyond filing issues. This is intended for the internal audience.

Reply:
I noticed that some content/sentences in these markdown files could be interpreted as encouraging external involvement beyond filing issues. Allowing to see how things evolve sounds good 👍.